src/shared/operator_cloud_client/intent_events_sender.go

```diff
@@ -82,7 +82,7 @@ func (ies *IntentEventsPeriodicReporter) Start(ctx context.Context) error {
 	go func() {
 		defer errorreporter.AutoNotify()
 		// Wait for caches to sync
-		ies.waitForCacheSync(ctx)
+		ies.k8sClusterManager.GetCache().WaitForCacheSync(ctx)
```

Comment:
Changed this to try once, as this waits until cache sync, and can't be retried (once it succeeds it returns immediately, once it fails it fails immediately - causes log spam)